web/src/main/kotlin/no/kartverket/matrikkel/bygning/routes/v1/bygning/BygningRoutes.kt

```diff
@@ -115,3 +178,34 @@ private fun Route.bruksenhetDoc() {
     }
 }
 
+private fun Route.bruksenhetEgenregistrertDoc() {
+    install(NotarizedRoute()) {
+        tags = setOf("Bygninger")
+        parameters = listOf(
+            Parameter(
+                name = "bygningId", `in` = Parameter.Location.path, schema = TypeDefinition.STRING,
+            ),
+            Parameter(
+                name = "bruksenhetId", `in` = Parameter.Location.path, schema = TypeDefinition.STRING,
+            ),
+        )
+
+        get = GetInfo.builder {
+            summary("Hent en bruksenhet med kun egenregistrert data")
+            description("Henter en bruksenhet, men kun egenregistrert data")
+
+            response {
+                responseCode(HttpStatusCode.OK)
+                responseType<BruksenhetSimpleResponse>()
+                description("Bruksenhet")
```

Comment:
Vet ikke helt hva, men burde vi hatt en egen beskrivelse av denne "simple" typen? For å skille mellom `BruksenhetResponse` og `BruksenhetSimpleResponse`
```suggestion
                description("Bruksenhet")
```